pornography-hosts

```diff
@@ -2761,6 +2761,7 @@
 0.0.0.0 cyberangels.org
 0.0.0.0 cybererotica.com
 0.0.0.0 cybereroticamobile.com
+0.0.0.0 cyberleaks.to
```

Comment:
you should add `www.$domain` as well

https://mypdns.org/my-privacy-dns/porn-records/-/issues/832#additional-requirements-for-dumb-hosts-files